### ISSUE TITLE
Typo in description of python.analysis.inlayHints.pytestParameters

### ISSUE THREAD

**Issue by SuperSodaSea** [microsoft/pylance-release#4576]
## Environment data

-   Language Server version: 2023.6.40
-   OS and version: Windows 10 Pro 2004
-   Python version (& distribution if applicable, e.g. Anaconda): 3.10.5

## Code Snippet

N/A

## Repro Steps

1. Open VS Code settings
2. Find python.analysis.inlayHints.pytestParameters

## Expected behavior

```python
def test_foo(my_fixture: 'str'):
    assert(my_fixture == 'foo')
```

## Actual behavior

![image](https://github.com/microsoft/pylance-release/assets/8724868/d2eb364f-8dbb-45de-88e5-87d3cdde537b)

## Logs

N/A

Yeah I know this is a dumb issue, but I hope we can have a correct code snippet in the description :)

**Comment by StellaHuang95** [comment]
Thanks for the issue! This will be fixed in the next version.

**Comment by heejaechang** [comment]
This issue has been fixed in prerelease version 2023.7.21, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2023721-12-July-2023-PreRelease)
